python/mxnet/gluon/contrib/estimator/estimator.py

```diff
@@ -202,13 +206,17 @@ def _add_default_training_metrics(self):
             self._train_metrics.append(metric_loss(loss_name))
 
         for metric in self._train_metrics:
-            metric.name = "training " + metric.name
+            metric.name = 'training ' + metric.name
```

Comment:
Please add a comment describing that this is needed for `LoggingHandler` to improve the logging output and to detect if it is in training or evaluation mode.

Reply:
I will add the comment here.

Reply:
The naming is not exactly hack. It already exists in the code for displaying name during logging. Anyway I have added comments to resolve the issue.